scala-api/src/main/scala/com/cloudera/livy/scalaapi/ScalaJobHandle.scala

```diff
@@ -79,8 +132,24 @@ class ScalaJobHandle[T] private[livy] (jobHandle: JobHandle[T]) extends Future[T
     })
   }
 
+  /**
+   *  Returns whether the job has already been completed with
+   *  a value or an exception.
+   *
+   *  $nonDeterministic
+   *
+   *  @return    `true` if the job is already completed, `false` otherwise
+   */
   override def isCompleted: Boolean = jobHandle.isDone
 
+  /**
+   *  The value of the job
```

Comment:
"The result value of the job."

Also, there are a few more places missing a period at the end of the sentence, please fix those.